### DISCUSSION TITLE
New Install on Windows -- No repository detected

### DISCUSSION THREAD

**Discussion by Hossy** [gitkraken/vscode-gitlens#3121] [Q&A]
I'm sure I'm missing something really simple here, but I couldn't find any Getting Started or install tips, so here goes...

I've been using SourceTree to manage all my Git repo activities, but would like to incorporate those into VSCode.  I just installed this extension and the first thing I noticed was, well, that I noticed nothing.  Nothing changed in any of the files I have open.  I did notice the new GitLens icon on the left bar.  I clicked it and it says **No repository detected** and I need to open a folder containing a git repository.  Well, I have over 50 of them open.  Any idea what I need to check next?

**Comment by jkelroy** [comment]
Hello @Hossy, how did you open the repositories in VS Code? If you look at the Source Control Sidebar in VS Code, does it also prompt you to open a repository?

If you open a repository from the GitLens prompt, are you able to work with it as expected?

**Reply by Hossy** [reply]
I've never used the Source Control Sidebar.  It is prompting me to install Git for Windows, probably because it's not seeing where git is currently installed either using the embedded git from SourceTree, from the MINIGW32, or from WSL.  I can't tell if GitLens is running into the same problem, though (although, presumably yes).  I can't get any debug output from GitLens.  Do I have to install a fourth copy of git (for Windows) or can I have VSCode/GitLens use one of the existing copies I have?

**Reply by jkelroy** [reply]
To resolve this, you should either be able to add Git to your PATH or point to the git.path in the Visual Studio Code settings - ctrl + shift + P > _Preferences: Open Default Settings _(JSON)__ > set the _"git.path"_.
